tests/Profile/BaseProfileTest.php

```diff
@@ -69,7 +69,6 @@ public function setup(): void
      * @covers ::__construct
      * @covers ::getProfileAttribute
      * @covers ::setAttributesMap
-     * @covers ::getAttributesByName
```

Comment:
Should this line have been removed?

Reply:
1. We have this method here - Yoti\Profile\Util\Attribute\AttributeListConverter and tests in Yoti\Test\Profile\Util\Attribute\AttributeListConverterTest.

2.We have these methods in another classes also, instead set we use construct method and convert everything before call constructor in class above.

3. All these methods are covered by tests, we have 100% )

4. Should be removed - David created special test for this method on 95 line.